pokemon_battle_arena/src/db/moves.rs

```diff
@@ -480,7 +478,7 @@ impl Technique {
         0
     }
 
-    pub fn get_effectivity_map(&self) -> HashMap<enums::types, i8> {
+    pub fn get_effectivity_map(&self) -> HashMap<enums::Types, i8> {
         self.clone().effectivity_map.unwrap()
     }
```

Comment:
Are you sure you want to return the `HashMap` by value? Why do you need that? I guess returning a reference is sufficient. Again: avoiding clone()s.